install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -741,9 +741,11 @@ label.
 
 This means two things for unprivileged containers:
 
-- The volume will be given a `type` which is accessible by unprivileged
-containers. This type is usually `container_file_t`, which treats volumes as container content. Previously, the label specified was `svirt_sandbox_file_t`. This label
-is no longer used due to security concerns.
+. The volume will be given a type which is accessible by unprivileged containers.
```

Comment:
Change `-` to `.` changed the format. It should be the same format as https://github.com/openshift/openshift-docs/pull/10716/files#diff-ca86745e66ae30b19d378fbbaef59fa4L747